battleship/board.py

```diff
@@ -6,22 +6,19 @@ class Board(object):
     def __init__(self):
         self.board = [[OCEAN for _ in range(10)] for _ in range(10)]
         self.hit_ships = []
-        self.attacks = []
 
     def __getitem__(self, i):
         return self.board[i]
 
     def attack(self, row, col):
-        if (row, col) in self.attacks:
-            raise ValueError("Already attacked these coordinates: {}:{}".format(row, col))
-
         spot = self.board[row][col]
-        self.attacks.append((row, col))
         if spot != OCEAN:
-            self.board[row][col] = 'X'
+            self.board[row][col] = 'H'
             self.hit_ships.append(spot)
             return "Hit! {}:{}".format(row, col)
-        return "Miss. {}:{}".format(row, col)
+        else:
+            self.board[row][col] = 'M'
+            return "Miss. {}:{}".format(row, col)
 
```

Comment:
What will happen if someone accidentally tries to attack a spot that has already been attacked with this code if it's a current 'M' it will change to 'H' and look like they hit something.